### ISSUE TITLE
A question about the implementation of TreeLSTM

### ISSUE THREAD

**Issue by speedcell4** [facebookresearch/latent-treelstm#5]
https://github.com/facebookresearch/latent-treelstm/blob/master/modules/BinaryTreeLstmCell.py#L33

I guess using `c.tanh_()` instead of `c.tanh()` will change the value of `c` in place. So what you actually return is `tanh(c)` instead of `c`. 
Is this a bug?

**Comment by serhii-havrylov** [comment]
Hi @speedcell4, 

Indeed, you are right. The cell state is going to be changed after c.tanh_(). Even though I think it is not going to change results dramatically, I will try to rerun the experiments to make sure it is the case.

**Comment by speedcell4** [comment]
> Even though I think it is not going to change results dramatically

Yes, I agree with you, whereas could you please let me know the new results after you finish experiments? I am following this work, these results are important to me.
Thank you~
